crates/rpc-types-eth/src/block.rs

```diff
@@ -149,6 +149,60 @@ impl Header {
     }
 }
 
+impl TryFrom<Header> for alloy_consensus::Header {
+    type Error = String;
+    fn try_from(value: Header) -> Result<Self, Self::Error> {
+        let Header {
+            hash: _hash,
+            parent_hash,
+            uncles_hash,
+            miner,
+            state_root,
+            transactions_root,
+            receipts_root,
+            logs_bloom,
+            difficulty,
+            number,
+            gas_limit,
+            gas_used,
+            timestamp,
+            total_difficulty: _total_difficulty,
```

Comment:
can we group this and hash and separate with a `// not included in header` comment?